### PR TITLE
Remove redundant text in feature request issue form

### PR DESCRIPTION
**Pull request by opusforlife2**
### Pull Request Checklist

- [x] Pull request is based on the develop branch
- [x] Pull request includes a new file under ./changelog.d. See https://github.com/vector-im/element-android/blob/develop/CONTRIBUTING.md#changelog
- [x] You've made a self review of your PR

This is a similar PR to #4076. It removes a redundant sentence.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/enhancement.yml [hunk 1]
```diff
@@ -10,10 +10,10 @@ body:
     id: usecase
     attributes:
       label: Your use case
-      description: What would you like to be able to do? Please feel welcome to include screenshots or mock ups.
+      description: Please feel welcome to include screenshots or mock ups.
       placeholder: Tell us what you would like to do!
       value: |
-        #### What would you like to do?
+        #### What would you like to be able to do?
```
Comment (kittykat) [inline .github/ISSUE_TEMPLATE/enhancement.yml:16]: This change isn't needed, the old string works and I'd like to keep it short

**Review by kittykat — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/enhancement.yml (hunk 1 above)
Comment (opusforlife2) [reply .github/ISSUE_TEMPLATE/enhancement.yml]: Done.

**Review by kittykat — APPROVED** [review]

**Comment by kittykat** [comment]
@opusforlife2 could you please add a signoff before we merge this? (`git commit --amend --signoff` and force push to your branch)

**Comment by bmarty** [comment]
> Here's the self review. I really don't get the point of this.

A self review is basically open https://github.com/vector-im/element-android/pull/4257/files and check that everything looks fine. (Stupid) mistakes can then be fixed by the submitter before requesting a review from someone else.

**Comment by opusforlife2** [comment]
@kittykat I read the contributing guide before opening my first PR, which says that you require legal names, and anonymous contributions aren't allowed, but I saw several merged PRs which didn't have a sign off, so I assumed that for minor changes, this isn't much of an issue.

If this is a blocker then please let me know, because I wish to stay anonymous (hence no sign offs).

(Also, I only use Github's web interface to create PRs, and I don't know if it lets you sign off on PRs, anyway.)

**Comment by kittykat** [comment]
> @kittykat I read the contributing guide before opening my first PR, which says that you require legal names, and anonymous contributions aren't allowed, but I saw several merged PRs which didn't have a sign off, so I assumed that for minor changes, this isn't much of an issue.
> 
> If this is a blocker then please let me know, because I wish to stay anonymous (hence no sign offs).

That's completely understandable. I will double check on policy and get back to you.

> (Also, I only use Github's web interface to create PRs, and I don't know if it lets you sign off on PRs, anyway.)

(Technically you'd just need to add a line to your PR/commits which says "Signed-off-by: name <email>")

**Comment by opusforlife2** [comment]
Oh, so "Signed off by: opusforlife2" is fine?

**Comment by bmarty** [comment]
> but I saw several merged PRs which didn't have a sign off

If the PR is not coming by someone from the Element team this is not normal. Do you have an example in mind?

And yes, I assume that https://github.com/vector-im/element-android/pull/4257#issuecomment-969136707 is enough.

**Comment by opusforlife2** [comment]
Ohkay. If they're exempt, then it is likely I saw some team member PRs and made an incorrect assumption. Sorry for the confusion.

> And yes, I assume that #4257 (comment) is enough.

Ummm. Can I sign off on future commits? I can't edit existing commits on Github.

**Pull request merged**